### DISCUSSION TITLE
docker环境下关于工作区开发的问题

### DISCUSSION THREAD

**Discussion by NFLSCode** [koishijs/koishi#915] [Q&A] [answered]
我通过 `docker pull koishijs/koishi` 和 `docker run -p 5140:5140 koishijs/koishi` 创建了容器。

接下来我尝试按照[文档](https://koishi.chat/manual/advanced/workspace.html)操作，但是当我运行

```
npm run setup example
```
的时候，npm 提示不存在

```
/koishi # npm run setup example
npm ERR! Missing script: "setup"
npm ERR! 
npm ERR! To see a list of scripts, run:
npm ERR!   npm run

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2023-01-10T04_50_04_130Z-debug-0.log
```

进一步的，我执行了 `npm run`，发现只有 start 一个脚本。

请问如何解决？

**Comment by shigma** [comment, marked as answer]
工作区开发的前置条件是你[创建模板项目](https://koishi.chat/manual/starter/boilerplate.html)。而官方的 docker 镜像是一个生产环境而非开发环境，是不能用于开发的。
